x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/constants/parameters_definition.tsx

```diff
@@ -224,28 +237,37 @@ export const PARAMETERS_DEFINITION = {
         },
       },
     },
+    schema: Joi.object().keys({
+      min: Joi.number(),
+      max: Joi.number(),
+      min_segment_size: Joi.number(),
+    }),
   },
   coerce: {
     fieldConfig: {
       defaultValue: true,
     },
+    schema: Joi.boolean().strict(),
   },
   coerce_shape: {
     fieldConfig: {
       defaultValue: false,
     },
+    schema: Joi.boolean().strict(),
   },
   ignore_malformed: {
     fieldConfig: {
       defaultValue: false,
     },
+    schema: Joi.boolean().strict(),
   },
   null_value: {
     fieldConfig: {
       defaultValue: '',
       type: FIELD_TYPES.TEXT,
       label: nullValueLabel,
     },
+    schema: Joi.string(),
   },
   null_value_ip: {
```

Comment:
I think this is missing `schema: Joi.string().ip()`

Reply:
As discussed over zoom for now we are not going to validate "special" null values